notebooks/community/sdk/SDK_BigQuery_Custom_Container_Training.ipynb

```diff
@@ -58,8 +58,15 @@
       },
```

Comment:
<p>Remove the whole paragraph -- not relevant.</p><p> </p><p>Add 1 sentence description what notebook is about.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>